src/Controller/ArticlesController.php

```diff
@@ -1170,59 +1175,93 @@ private function articlePageArguments(Request $request, string $id, int $version
                     ->toArray();
 
                 $latestVersion = $articleVersions[count($articleVersions) - 1]->getVersion();
-
-                $otherLinks = [];
                 if (isset($eraArticle['display']) && $item->getVersion() === $latestVersion) {
-                    $otherLinks[] = new Link(
+                    $links[] = ViewModel\TabbedNavigationLink::fromLink(new Link(
                         'Executable code',
                         $this->get('router')->generate('article-era', ['id' => $item->getId()])
+                    ));
+                }
+
+                if ($hasPeerReview) {
+                    $links[] = ViewModel\TabbedNavigationLink::fromLink(
+                                        new Link('Peer Review', $this->generatePath($history, $item->getVersion(), null, 'content'))
+                                );
+                }
+
+                if ($item instanceof ArticleVoR) {
+                    $sideBySideUrl = rtrim($this->getParameter('side_by_side_view_url'), '/').'/'.$item->getId();
+                    $links[] = ViewModel\TabbedNavigationLink::fromLink(
+                        new Link('Side by side', $sideBySideUrl), null, true
                     );
                 }
 
-                return new ViewSelector(
-                    new Link('Article', $this->generatePath($history, $item->getVersion(), null, 'content')),
-                    array_values(array_filter(array_map(function (ViewModel $viewModel) {
+                return new TabbedNavigation($links);
+            });
+    }
+
+    private function createJumpMenu(PromiseInterface $item, PromiseInterface $isMagazine, PromiseInterface $hasFigures, bool $isFiguresPage, PromiseInterface $history, PromiseInterface $sections, array $eraArticle) : PromiseInterface
+    {
+        return all(['item' => $item, 'isMagazine' => $isMagazine, 'hasFigures' => $hasFigures, 'history' => $history, 'sections' => $sections])
+            ->then(function (array $sections) use ($isFiguresPage, $eraArticle) {
+                if ($sections['isMagazine']) {
+                    return null;
+                }
+
+                $hasFigures = $sections['hasFigures'];
+                $sections = $sections['sections'];
+
+                $sections = array_filter($sections, Callback::isInstanceOf(ArticleSection::class));
+
+                if (count($sections) < 2) {
+                    if (!$hasFigures) {
+                        return null;
+                    }
+
+                    $sections = [];
+                }
+
+                 $links = [];
+
+                return new JumpMenu(
+                    array_merge($links, array_values(array_filter(array_map(function (ViewModel $viewModel) {
                         if ($viewModel instanceof ArticleSection) {
                             return new Link($viewModel['title'], '#'.$viewModel['id']);
                         }
-
+                
```

Comment:
nitpick: remove whitespace from start of line. Ideally this would be picked up by phpcs.